cvat/apps/engine/migrations/0028_labelcolor.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.13 on 2020-08-11 11:26
+from django.db import migrations, models
+from cvat.apps.dataset_manager.formats.utils import get_color_from_label_name, normalize_label, rgb2hex
+
+def alter_label_colors(apps, schema_editor):
+    Label = apps.get_model('engine', 'Label')
+    Task = apps.get_model('engine', 'Task')
+
+    for task in Task.objects.all():
+        labels = Label.objects.filter(task_id=task.id)
+        normalized_names = [normalize_label(l.name) for l in labels]
```

Comment:
I see the code in 3 places. Need to remove code cloning because the logic isn't trivial.

Will `Road` and `road` have the same color? Is it OK to have the same color for different classes?

Reply:
@nmanovic, sorry for a long wait. Realized that default python hash function has a random seed every run.

> I see the code in 3 places. Need to remove code cloning because the logic isn't trivial.

Done.

> Will Road and road have the same color? Is it OK to have the same color for different classes?

If that labels will appear in the task only once they will have the same color. But if label will appear in the task more than one time, for the first label color will be taken from the color map and color will be calculated with hash for others.